.storybook/preview.js

```diff
@@ -1,4 +1,62 @@
 import {addons} from '@storybook/addons'
+import {ThemeProvider, themeGet, theme} from '../src'
+import {createGlobalStyle} from 'styled-components'
+
+// set global theme styles for each story
+const GlobalStyle = createGlobalStyle`
+  body {
+    background-color: ${themeGet('colors.bg.primary')};
+    color: ${themeGet('colors.text.primary')};
+  }
+`
+export const globalTypes = {
+  colorMode: {
+    name: 'Color mode',
+    description: 'Color mode (day, night, auto)',
+    defaultValue: 'day',
+    toolbar: {
+      icon: 'paintbrush',
+      // array of colorMode items
+      items: Object.keys(theme.colorSchemes),
```

Comment:
This should stay hard-coded 😅 

"color modes" and "color schemes" are _different_ (but related) ideas in Primer React.

```suggestion
      items: ['day', 'night', 'auto'],
```